### DISCUSSION TITLE
Is Account ID unique across all instances?

### DISCUSSION THREAD

**Discussion by sudhagar-mr-17051** [mastodon/mastodon#25938] [Q&A]
Are mastodon account ID's unique across all instances or there is possibility of replication of same account ID in multiple mastodon instances,

Kindly clarify and thanks in advance!

**Comment by Gargron** [comment]
Numeric IDs are not unique. Username and domain are unique.

**Reply by sudhagar-mr-17051** [reply]
When you say username, are you referring abc@mastodon.social and abc@mastodon.world are unique, but we have to use usernames as full username with domain which makes it unique, am I right?
